usaspending_api/api_contracts/contracts/v2/recipient/count.md

```diff
@@ -0,0 +1,49 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Recipient Duns [/api/v2/recipient/duns/count/]
+
+These endpoints are used to power USAspending.gov's recipient profile pages.
+
+## POST 
+
+This endpoint returns the count of recipients given an award_type and keyword string.
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object"
+            }
+
+    + Attributes (object)
+        + `keyword` (optional, string)
+            The keyword results are filtered by. Searches on name and DUNS.
+        + `award_type` (optional, enum[string])
+            The award type results are filtered by.
+            + Default: `all`
+            + Members
+                + `all`
+                + `contracts`
+                + `grants`
+                + `loans`
+                + `direct_payments`
+                + `other_financial_assistance`
```

Comment:
Should IDV be added here as an award type, even though we don't show it on the recipients listings page?
That may have been an oversight when introducing IDV, that we didn't add it to the page.

Reply:
I don't see any IDV in this table. Adding it may require some changes to the ETL